tests/data/scripts_pythondistdeps/pyreq2rpm.tests-2020.04.07.024dab0-py3.9.egg-info/requires.txt

```diff
@@ -0,0 +1,95 @@
+# Taken from pyreq2rpm, removed tests that are expected to fail
+foobar0~=2.4.8
+foobar1~=2.4.8.0
+foobar2~=2.4.8.1
+
+foobar4~=2.0
+
+
+foobar7~=2.4.8b5
+foobar8~=2.0.0b5
+foobar9~=2.4.8.post1
+foobar10~=2.0.post1
+foobar11==2.4.8
+foobar12==2.4.8.0
+foobar13==2.4.8.1
+foobar14==2.4.8.*
+foobar15==2.0
+foobar16==2
+foobar17==2.*
+foobar18==2.4.8b5
+foobar19==2.0.0b5
+foobar20==2.4.8.post1
+foobar21==2.0.post1
+foobar22===2.4.8
+foobar23===2.4.8.0
+foobar24===2.4.8.1
+
+foobar26===2.0
+foobar27===2
+
+foobar29===2.4.8b5
+foobar30===2.0.0b5
+foobar31===2.4.8.post1
+foobar32===2.0.post1
```

Comment:
Add dev versuon requirement somewhere here to also test requires, not only provides.

Reply:
+1